PyRoles.py

```diff
@@ -44,36 +45,40 @@ def checkDuplicate(photo):
     hashPhoto = hashlib.md5(data_foto).hexdigest() 
     hashTable = json.loads(data)
 
+
+
     if (hashPhoto in hashTable.values()):
         return True
+    
     else:
+        
+        hashTable[hashPhoto] = hashPhoto
+        with open('hash_table.txt', 'w') as file:
+            file.write(json.dumps(hashTable))
```

Comment:
Acho que ficaria legal uma quebra de linha ao escrever no arquivo..
To vendo aqui no server e tá criando uma linha gigante com os dados..